rotkehlchen/accounting/accountant.py

```diff
@@ -280,6 +283,10 @@ def _process_event(
             )
             return 1, prev_time
 
+        if eth2_taxable_after_withdrawal_enabled is True and A_ETH2 in event_assets:
```

Comment:
So `eth2_taxable_after_withdrawal_only` or `eth_staking_taxable_after_withdrawal_enabled`(following the initial suggestion by Left)

Reply:
Ah, nope, now I think that current name is better than my suggestion. So for confusing you :disappointed: .

When I was suggesting to add `only` I was not thinking that with the `False` value it would mean that eth2 is taxable not only after withdrawal but at some other point too.